zebra-chain/src/transaction/serialize.rs

```diff
@@ -285,6 +300,26 @@ impl ZcashDeserialize for Transaction {
                     joinsplit_data,
                 })
             }
+            (5, false) => {
+                let id = reader.read_u32::<LittleEndian>()?;
+                if id != TX_V5_VERSION_GROUP_ID {
+                    return Err(SerializationError::Parse("expected TX_V5_VERSION_GROUP_ID"));
+                }
+                let inputs = Vec::zcash_deserialize(&mut reader)?;
```

Comment:
Similarly, the order of these fields should match the draft v5 transaction spec.